src/sets.jl

```diff
@@ -263,6 +321,18 @@ end
 dual_set(s::RotatedSecondOrderCone) = copy(s)
 dual_set_type(::Type{RotatedSecondOrderCone}) = RotatedSecondOrderCone
 
+function distance_to_set(v::AbstractVector{<:Real}, s::RotatedSecondOrderCone)
+    _check_dimension(v, s)
+    t = v[1]
+    u = v[2]
+    xs = v[3:end]
+    return [
```

Comment:
Why return a scalar for `SecondOrderCone` and a vector for `RotatedSecondOrderCone`?